.github/workflows/build-project.yaml

```diff
@@ -365,6 +365,15 @@ jobs:
           target: x64
           config: ${{ needs.check-event.outputs.config }}
 
+      - name: Run PVS-Studio Analysis
```

Comment:
I'd rather have us use a new `analyze-project` workflow that runs in parallel to the build workflow. This one could then be extended to use the clang analyse build or ASAN and other code quality checkers.